crates/services/gas_price_service/src/v1/da_source_service/service.rs

```diff
@@ -44,7 +44,7 @@ pub struct DaSourceService<Source> {
     source: Source,
     shared_state: SharedState,
     latest_l2_height: Arc<Mutex<BlockHeight>>,
-    recorded_height: Option<BlockHeight>,
+    recorded_height: BlockHeight,
```

Comment:
Could you please add comments? It not clear at first glance what's the difference between `latest_l2_height` and `recorded_height`. Especially that in other places we also use `latest_recorded_height` which sounds like a mix of both :)

Reply:
let me know if this works: 2fe4efe8e9